geode-core/src/main/java/org/apache/geode/internal/process/MBeanProcessController.java

```diff
@@ -260,6 +265,7 @@ private JMXServiceURL getJMXServiceURL() throws AttachNotSupportedException, IOE
         // need to load the management-agent and get the address
 
         final String javaHome = vm.getSystemProperties().getProperty("java.home");
+        assertState(StringUtils.isNotBlank(javaHome), CliStrings.JAVA_HOME_NOT_FOUND_ERROR_MESSAGE);
```

Comment:
You're right... I was trying to find a place for the `JAVA_HOME_NOT_FOUND_ERROR_MESSAGE` variable, but it looks like it would change some behavior if inserted here. I'll just roll back the changes on this file and delete `JAVA_HOME_NOT_FOUND_ERROR_MESSAGE`. Thanks for your feedback!